doc/grahom.xml

```diff
@@ -274,14 +274,35 @@ Transformation( [ 3, 1, 5, 4, 5 ] )
     <Ref Oper="HomomorphismsDigraphsRepresentatives"/>, except they only return
     <E>injective</E> homomorphisms.
     <Example><![CDATA[
-gap> gr1 := ChainDigraph(3);;
-gap> gr2 := Digraph([[3, 5], [2], [3, 1], [], [4]]);
-<immutable digraph with 5 vertices, 6 edges>
-gap> MonomorphismsDigraphs(gr1, gr2);
-[ Transformation( [ 1, 5, 4, 4, 5 ] ), 
-  Transformation( [ 3, 1, 5, 4, 5 ] ) ]
-gap> MonomorphismsDigraphsRepresentatives(gr1, CompleteDigraph(3));
-[ Transformation( [ 2, 1 ] ) ]
+gap> SubdigraphsMonomorphisms(CompleteBipartiteDigraph(2, 2),
+> CompleteDigraph(4));
+[ Transformation( [ 1, 3, 2 ] ), Transformation( [ 2, 3, 1 ] ),
+  Transformation( [ 3, 4, 2, 1 ] ) ]
+]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="SubdigraphsMonomorphisms">
+<ManSection>
+  <Oper Name="SubdigraphsMonomorphisms" Arg="digraph1, digraph2"/>
+  <Oper Name="SubdigraphsMonomorphismsRepresentatives" Arg="digraph1, digraph2"/>
+  <Returns>A list of transformations.</Returns>
+  <Description>
+    These operations behave the same as <Ref Oper="HomomorphismsDigraphs"/> and
+    <Ref Oper="HomomorphismsDigraphsRepresentatives"/>, except they only return
+    <E>injective</E> homomorphisms with the following property: the (not
+    necessarily induced) subdigraphs defined by the images of these
+    monomorphisms are all of the subdigraphs of <A>digraph2</A> that are
+    isomorphic to <A>digraph1</A>. Note that the subdigraphs of the previous
+    sentence are those obtained by applying the corresponding monomorphism to
+    the vertices and the edges of <A>digraph1</A>, and are therefore possibly
+    strictly contained in the induced subdigraph on the same vertex set.
+    <Example><![CDATA[
+gap> SubdigraphsMonomorphisms(CompleteBipartiteDigraph(2, 2),
+> CompleteDigraph(4));
+[ Transformation( [ 1, 3, 2 ] ), Transformation( [ 2, 3, 1 ] ),
+  Transformation( [ 3, 4, 2, 1 ] ) ]
```

Comment:
Maybe it would be nice to have also an example for SubdigraphsMonomorphismsRepresentatives, such that the difference between the methods is clear.

Reply:
Good point, doing it now